docs/source/faq.rst

```diff
@@ -65,25 +193,3 @@ stability. If you would like to build from source but do not require the
 latest feature set, use the
 `stable <https://github.com/facebookresearch/CompilerGym/commits/stable>`_
 branch which lags to the latest release with hotfixes.
-
-
-I want to add a new program representation / reward signal. How do I do that?
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-If your reward or observation is a transformation of an existing space, consider
-using the :mod:`compiler_gym.wrappers` module to define a wrapper that performs
-the translation from the base space.
-
-If your reward or observation requires combining multiple existing spaces,
-consider using :meth:`add_derived_space()
-<compiler_gym.views.ObservationView.add_derived_space>` or :meth:`add_space()
-<compiler_gym.views.RewardView.add_space>`.
-
-If you require modifying the underlying compiler service implementation, fork
-this project and build it from source (see `installation
-<https://github.com/facebookresearch/CompilerGym/blob/development/INSTALL.md>`_).
-Then modify the service implementation for the compiler that you are interested
-in. The service codebase is located at
-:code:`compiler_gym/envs/$COMPILER/service`, where :code:`$COMPILER` is the name
-of the compiler service you would wish to modify, e.g. llvm. Once done, send us
-a pull request!
```

Comment:
Why is that removed? Is it no longer valid?

Reply:
I merged it into a more general "I want to modify a CompilerGym environment, where do I start?", which covers other kinds of changes